2019-09-04 23:13:13,101 ERROR: Exception on /favicon.ico [GET] [in c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 1947, in full_dispatch_request
    rv = self.preprocess_request()
  File "c:\users\tomas\appdata\local\programs\python\python36\lib\site-packages\flask\app.py", line 2241, in preprocess_request
    rv = func()
  File "C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\routes.py", line 16, in before_request
    g.locale = str(get_locale())
NameError: name 'g' is not defined
2019-09-04 23:14:00,635 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-04 23:36:45,007 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:21:16,127 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:21:41,214 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:25:46,648 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:29:24,098 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:35:29,852 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-05 21:45:10,854 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-06 18:12:58,157 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-06 18:13:11,121 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-06 20:57:41,626 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 14:14:21,522 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 14:56:50,938 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 14:57:54,707 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 15:05:41,668 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 15:34:41,749 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 15:37:10,001 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 15:37:45,127 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 15:46:03,793 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 16:03:36,102 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 16:05:32,990 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 16:07:22,464 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 17:52:21,437 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 17:52:42,601 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:03:40,801 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:06:38,359 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:09:58,986 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:10:15,123 ERROR: Exception on /translate [POST] [in c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\routes.py", line 203, in translate_text
    request.form['dest_language'])})
TypeError: translate() missing 1 required positional argument: 'dest_language'
2019-09-07 18:10:21,635 ERROR: Exception on /translate [POST] [in c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\tomas\desktop\python\website-with-flask\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\routes.py", line 203, in translate_text
    request.form['dest_language'])})
TypeError: translate() missing 1 required positional argument: 'dest_language'
2019-09-07 18:12:31,114 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:16:51,951 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:28:01,355 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:30:40,424 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:31:45,541 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:33:10,699 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
2019-09-07 18:33:31,733 INFO: Microblog startup [in C:\Users\tomas\Desktop\Python\Website-with-Flask\microblog\app\__init__.py:52]
